Dockerfile

```
@@ -0,0 +1,20 @@
+FROM --platform=linux/amd64 clux/muslrust:stable as builder
+ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
+RUN mkdir /app
+WORKDIR /app
+COPY . .
+RUN rustup target add x86_64-unknown-linux-musl
+RUN cargo build --release --target x86_64-unknown-linux-musl
+RUN strip /app/target/x86_64-unknown-linux-musl/release/rindexer_cli
+
+FROM --platform=linux/amd64 node:16 as node-builder
```

```suggestion
FROM --platform=linux/amd64 node:lts as node-builder
```

Node.js v16 is no longer maintained and doesn't receive security updates
https://endoflife.date/nodejs

---

yeah I've seen that it was used inside the package.json for building the graphql binary but you're right we should bump it